src/trust/index.ts

```diff
@@ -1,443 +1,5 @@
-import { decode, verify } from "./utils";
-import { decode as decodeJwt } from "@pagopa/io-react-native-jwt";
-import {
-  CredentialIssuerEntityConfiguration,
-  EntityConfiguration,
-  EntityStatement,
-  FederationListResponse,
-  RelyingPartyEntityConfiguration,
-  TrustAnchorEntityConfiguration,
-  WalletProviderEntityConfiguration,
-} from "./types";
-import { renewTrustChain, validateTrustChain } from "./chain";
-import { hasStatusOrThrow } from "../utils/misc";
-import type { JWK } from "../utils/jwk";
-import {
-  BuildTrustChainError,
-  FederationListParseError,
-  MissingFederationFetchEndpointError,
-  RelyingPartyNotAuthorizedError,
-  TrustAnchorKidMissingError,
-} from "./errors";
-import type { X509CertificateOptions } from "@pagopa/io-react-native-crypto";
+import * as Build from "./build-chain";
+import * as Verify from "./verify-chain";
+import * as Errors from "./errors";
 
-export type {
```

Comment:
Is it intentional we are not exporting this types anymore?

Reply:
Thanks for the catch, I forgot to re-export those types. For consistency, I exported the whole file. Fixed here https://github.com/pagopa/io-react-native-wallet/pull/247/commits/a11acc3bef8cb80c53a09e262789040ac9bb2a3b, thanks!